README.md

```diff
@@ -47,6 +54,18 @@ whatever you selected above (can also be accessed from the `Settings` >
 If you do not see the training modules folders in the kitematic folder, raise
 your hand.
 
+8. Go to Docker preferences.
```

Comment:
This was addressed in AlexsLemonade/training-modules#51. There are now steps in [`docker-install/README.md`](https://github.com/AlexsLemonade/training-modules/blob/master/docker-install/README.md) for Mac and Windows 10 (steps 5&6 and 6&7, respectively).

Reply:
This isn't the bit that is new. This is a github mix up, maybe I didn't merge to master. It's the permissions bit above that we need to add.